eng/Version.Details.xml

```diff
@@ -95,9 +95,9 @@
       <Uri>https://github.com/nuget/nuget.client</Uri>
       <Sha>c0d441fc5e99e6765c9e0b2f77de162b9866b305</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.11.0-3.24352.5">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.11.0-3.24378.3">
```

Comment:
@f-alizada should we pull the update to 17.12 out of your PR? It should go in here too (and then get rid of all these nonsense extra feeds).